### DISCUSSION TITLE
How do I close inline assist lines?

### DISCUSSION THREAD

**Discussion by heyAyushh** [zed-industries/zed#10313] [Feature Requests]
![Screenshot 2024-04-09 at 4 21 10 PM](https://github.com/zed-industries/zed/assets/17379963/26211b13-7cfb-4645-9fe8-5da03a99ce94)

**Comment by JosephTLyons** [comment]
Hey @heyAyushh, try `esc`. We probably should have some UI around closing these as well.

**Comment by heyAyushh** [comment]
thankyou so much, yea UI to close that makes sense
